gen2-yolo/yolo-segmentation/README.md

```diff
@@ -0,0 +1,75 @@
+# YOLO segmentation with decoding on host
+
+![On-host decoding YOLO segmentation in OAK](docs/oak_segmentation_example.gif)
+
+## Pre-requisites
+
+### 1. Install requirements
+
+```bash=
+python3 -m pip install -r requirements.txt
+```
+
+### 2. Convert/export models to ONNX format
+
+You can either train your custom model or try directly with a model trained on the COCO dataset. The latter is the case handled in this experiment.
+
+#### **YOLOv5**
+
+**1. Installation:**
+```bash=
+git clone https://github.com/ultralytics/yolov5.git
+cd yolov5/
+pip install -r requirements.txt
+```
+
+**2. Export:**
+```bash=
+python3 export.py --weights yolov5{n,s}-seg.pt --include onnx --imgsz 640 --opset 16 --simplify
+```
+
+#### **YOLOv8** and **YOLO11**
+
+**1. Installation:**
+```bash=
+pip install ultralytics
+```
+
+**2. Export:**
+```bash=
+yolo export model=yolov8{n,s}-seg.pt format=onnx imgsz=640 half=True dynamic=False simplify=True batch=1
+yolo export model=yolov11{n,s}-seg.pt format=onnx imgsz=640 half=True dynamic=False simplify=True batch=1
+```
+
+#### **YOLOv9**
```

Comment:
In this case, I prefer to use the original YOLOv9 repo.

Reply:
Is there any specific reason for that? I am only asking because this creates more dependencies than using the Ultralytics version for this one as well.

Reply:
Just because YOLOv9 does not belong to Ultralytics, so I assumed it would be better to use the original repo.